tests/test_http_request.py

```diff
@@ -19,6 +19,21 @@ def test_formdata_type(self):
         self.assertRaises(ValueError, _get_inputs, "", 5, "", "", "")
 
 
+class FormTest(unittest.TestCase):
+    # <input type="hidden" name="test" value="val1">
```

Comment:
To keep the format of the tests written by @jenniferlindberg, "# GROUP 12 ADDED TEST CASE" should be added as comment to the implemented test. Potentially the current comment could be replaced?

Reply:
yeah, makes sense! thanks